### PR TITLE
add devanooga to groups page

### PR DESCRIPTION
**Pull request by DigTK**

### CODE REVIEW

File: _data/groups.yml [hunk 1]
```diff
@@ -41,3 +41,10 @@
     urls:
       -   name: website
           url: http://www.meetup.com/Chattanooga-Python-User-Group/
+-   name: Devanooga
+    github: devanooga
+    code-of-conduct: http://devanooga.com/code-of-conduct/
+    gravatar-id: 09ce08e351a3affde0ffe7b6b74f2789
+    urls:
+        -   name: website
+            url: http://devanooga.com/
```
Comment (agarzola) [inline _data/groups.yml]: @DigTK devanooga supports `https`. I suggest updating the protocol here.

Comment (DigTK) [reply _data/groups.yml]: nice catch!

**Comment by ryanmaynard** [comment]
Any update on this, @dereklucas?

**Comment by camilopayan** [comment]
Extremely against this. Devanooga was created with the express purpose of creating an unprofessional environment that allows racist humor.

**Comment by brb3** [comment]
@camilopayan that statement is libelous. Please remove it. You aren't helping chadev or devanooga with that attitude.

**Comment by dereklucas** [comment]
This PR is rejected after discussion of this code of conduct, as it doesn’t meet ChaDev’s standards. Also, the Devanooga group has repeatedly been involved in activities and behavior that has been antagonistic to the ChaDev constituency.

**Pull request closed without merging**